Window.py

```diff
@@ -28,10 +28,12 @@ def redrawWindow(self):
 		self.window.blit(self.player1.image, (self.player1.xPosition, self.player1.yPosition))  #iscrtavanje naseg player1
 		self.window.blit(self.player2.image, (self.player2.xPosition, self.player2.yPosition))  # iscrtavanje naseg player2
 		self.window.blit(self.projectile.image, (self.projectile.xPosition, self.projectile.yPosition))
+		self.bubble.move_ball()
 		pygame.display.update()  # da bi se oni pojavili na ekranu
 
 
-	def runPlayers(self):
+	def runGame(self):
```

Comment:
I think we should have one method that starts whole game. In this method we have while loop, where we expect player movement, and where we should move ball also.